app/src/renderer/windows/index.html

```
@@ -1,5 +1,5 @@
 <!doctype html>
-<html>
+<html class="dark" data-theme="dark">
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)